platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -111,19 +111,35 @@ payment_observer:
   #
   enabled: false
 
-  ## @param: circle_url
-  ## @type: string
-  ## Circle api endpoint.
-  #
-  circle_url: https://api-sandbox.circle.com
-
-
-  ## @param: tracked_wallet
-  ## @type: string
-  ## The id of the wallet to track. 'all' indicates that events for all wallets are tracked.
-  #
-  tracked_wallet: all
+  network:
+    ## @param: type
+    ## @supported_values: `stellar`, `circle`
+    ## The type of payment network.
+    #
+    type: stellar
+    stellar:
+      ## @param: max_results
+      ## @type: int
+      ## The maximum number of results the Stellar Blockchain can return.
+      max_results: 200
+
+      ## @param: min_results
+      ## @type: int
+      ## The minimum number of results the Stellar Blockchain can return.
+      min_results: 1
```

Comment:
`Limit` is something that makes sense for paginated requests but not for observers. If you use a limit of 10, for instance, there's 100% chance of you will cause 429 (rate limit) issues and break the streamer completely.

Let's just hardcode it to `200`, since anything smaller than that increases the chances of bugs that we would eventually need to support.

Reply:
With that in mind, we don't need any Stellar-specific configuration for the streamer.